src/components/views/settings/SecureBackupPanel.tsx

```diff
@@ -100,12 +102,15 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
     private async checkKeyBackupStatus(): Promise<void> {
         this.getUpdatedDiagnostics();
         try {
-            const keyBackupResult = await MatrixClientPeg.safeGet().checkKeyBackup();
+            const keyBackupResult = await MatrixClientPeg.safeGet().getCrypto()!.checkKeyBackupAndEnable();
+            const activeBackupVersion =
+                (await MatrixClientPeg.safeGet().getCrypto()?.getActiveSessionBackupVersion()) ?? null;
```

Comment:
why `getCrypto()!` on one line and `getCrypto()?` on the next?

Reply:
Was not sure how to do it. The previous code was calling `client.xxx()` that was throwing an error if cryptoBackend was null. So I guess using ! is equivalent of previous code, but I am not confortable forcing unwrapping. It's not clear from the component design what happens if crypto is not enabled (though nowdays I don't think you can have a client without crypto)